easybuild/easyconfigs/g/Geant4/Geant4-10.5-intel-2017b.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Geant4'
+version = '10.5'
+
+homepage = 'http://geant4.cern.ch/'
+description = """Geant4 is a toolkit for the simulation of the passage of particles through matter.
+ Its areas of application include high energy, nuclear and accelerator physics, 
+ as well as studies in medical and space science."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['http://geant4.cern.ch/support/source']
+sources = ['%(namelower)s.%(version_major)s.0%(version_minor)s.tar.gz']
+checksums = ['4b05b4f7d50945459f8dbe287333b9efb772bd23d50920630d5454ec570b782d']
+
+builddependencies = [('CMake', '3.9.5')]
+dependencies = [
+    ('expat', '2.2.4'),
+    # recommended CLHEP version, see http://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.5.html#2.
+    ('CLHEP', '2.3.4.3'),
```

Comment:
Same remark about `CLHEP` version as in the other easyconfig file

Reply:
the new CLHEP is added.